recipes/r-taxonomycleanr/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set version = '1.6.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-taxonomycleanr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  git_url: https://github.com/EDIorg/taxonomyCleanr
```

Comment:
You should use this URL: https://github.com/EDIorg/taxonomyCleanr/archive/0814cf931e779526f5dff60307a4a6cddc1292d7.tar.gz otherwise you will always use the main brach.